app/graphql/types/system.rb

```diff
@@ -19,6 +19,10 @@ class System < Types::BaseObject
       argument :policy_id, ID, 'Filter results by policy or profile ID',
                required: false
     end
+    field :test_result_profiles, [::Types::Profile], null: true do
+      argument :policy_id, ID, 'Filter results by policy or profile ID',
+               required: false
```

Comment:
Maybe we should mention that this filters based on test results and not on which hosts are assigned to a policy:

`'Filter results tested against a policy or profile ID'` or similar.

Reply:
Changed.